models/value_set.go

```diff
@@ -47,12 +47,8 @@ type CodeSet struct {
 	Values []Concept
 }
 
-func (v *ValueSetMap) ReasonValueSetOid(codedValue CodedConcept, fieldOids map[string][]string) string {
-	return v.OidForCode(codedValue, fieldOids["REASON"])
-}
-
-func (v *ValueSetMap) OrdinalityValueSetOid(codedValue CodedConcept, fieldOids map[string][]string) string {
-	return v.OidForCode(codedValue, fieldOids["ORDINAL"])
+func (v *ValueSetMap) OidForCodeForField(codedValue CodedConcept, fieldOids map[string][]string, field string) string {
```

Comment:
Now that I look at this code, is there a way that we can just change all the calls to this method to use the one below it, OidForCode()? If the caller simply puts the field parameter into the map, the wrapper is unnecessary and OidForCodeForField() can go away.